### ISSUE TITLE
Support Spring HttpStatus

### ISSUE THREAD

**Issue by kakawait** [zalando/problem-spring-web#31]
Not really useful and not a great betterment (feel free to close it! I can really continue without, for sure)! But I could be cool that `AdviceTrait` support `org.springframework.http.HttpStatus` in addition to `javax.ws.rs.core.StatusType` (because we are speaking about `problem`**-spring-web**)

But I don't think you should support `org.springframework.http.HttpStatus` on `ProblemBuilder` (or other class in `Problem` package)  because is not `problem-spring-web` package I was just thinking about offer following `default`

```
ResponseEntity<Problem> create(final StatusType status, final Throwable throwable, final NativeWebRequest request) {
    ...
}

ResponseEntity<Problem> create(final HttpStatus status, final Throwable throwable, final NativeWebRequest request) {
    // Convert HttpStatus to StatusType
    return create(convert(status), throwable, request);
}
```

And same for other `default`

---

PS: I can submit a `PR` if you think is valuable

**Comment by whiskeysierra** [comment]
Makes sense.

> PS: I can submit a PR if you think is valuable

:+1: contributions are highly appreciated

**Comment by kakawait** [comment]
I will be in vacancies at the end of the week I don't think I will be able to push PR before it. I will work on it just after my return!

**Comment by whiskeysierra** [comment]
Fixed with #33
